proposal/package-importer.changes.md

```diff
@@ -1,3 +1,11 @@
+## Draft 1.5
+
+* Specify that Package Importers must not reject URL patterns that other Package
+  Importers may be able to canonicalize.
+
+* Specify that the Node Package Importer throws if `entryPointPath` is not
+    passed and `require.main.filename` is not available.
```

Comment:
Nit: fix indentation

Reply:
Fixed in 3ac835a42cf5a7f691d7a92d613cbf81eaf29531